Doc/library/datetime.rst

```diff
@@ -302,18 +302,18 @@ Supported operations:
 +--------------------------------+-----------------------------------------------+
 | Operation                      | Result                                        |
 +================================+===============================================+
-| ``t1 = t2 + t3``               | Sum of *t2* and *t3*. Afterwards *t1*-*t2* == |
-|                                | *t3* and *t1*-*t3* == *t2* are true. (1)      |
+| ``t1 = t2 + t3``               | Sum of *t2* and *t3*. Afterwards ``t1 - t2 == |
```

Comment:
Double backquotes should be used for variable names, as per [the devguide style recommendations](https://devguide.python.org/documentation/markup/#inline-markup).

```suggestion
| ``t1 = t2 + t3``               | Sum of ``t2`` and ``t3``. Afterwards ``t1 - t2 == |
```